02-proxmox-k3s-ansible-with-modules/main.tf

```diff
@@ -256,4 +256,17 @@ resource "local_file" "k3s_ansible_group_vars_all" {
     }
   )
   filename = "external/k3s-ansible/inventory/terraform/group_vars/all.yml"
+}
+    resource "null_resource" "example" {
+provisioner "remote-exec" {
+connection {
+  host = "192.168.0.14"
```

Comment:
We should not use hardcoded values here. Instead we need to refer from values from local